packages/EasyErrorHandler/src/Bridge/Laravel/Handler/Handler.php

```diff
@@ -0,0 +1,332 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Laravel\Handler;
+
+use Carbon\Carbon;
+use EoneoPay\ApiFormats\Bridge\Laravel\Traits\LaravelResponseTrait;
+use EoneoPay\ApiFormats\External\Interfaces\Psr7\Psr7FactoryInterface;
+use EoneoPay\ApiFormats\Interfaces\EncoderGuesserInterface;
+use EoneoPay\ApiFormats\Interfaces\EncoderInterface;
+use EoneoPay\Externals\Translator\Interfaces\TranslatorInterface;
+use EonX\EasyErrorHandler\Interfaces\LogLevelAwareExceptionInterface;
+use EonX\EasyErrorHandler\Interfaces\StatusCodeAwareExceptionInterface;
+use EonX\EasyErrorHandler\Interfaces\SubCodeAwareExceptionInterface;
+use EonX\EasyErrorHandler\Interfaces\TranslatableExceptionInterface;
+use EonX\EasyErrorHandler\Interfaces\ValidationExceptionInterface;
+use EonX\EasyLogging\Interfaces\LoggerInterface;
+use Exception;
+use Illuminate\Console\OutputStyle;
+use Illuminate\Contracts\Config\Repository;
+use Illuminate\Contracts\Debug\ExceptionHandler;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Support\Arr as IlluminateArr;
+use Symfony\Component\Console\Application as ConsoleApplication;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Handler implements ExceptionHandler
+{
+    use LaravelResponseTrait;
+
+    /**
+     * @var string
+     */
+    protected $defaultUserMessage = 'Oops, something went wrong.';
```

Comment:
I think that this default message should be a translation key so the app would have the ability to customise it without changing any code.

Reply:
We should also make it a constant, it would help for preloading so performance.

Reply:
done